openquake/hazardlib/gsim/boore_2014.py

```diff
@@ -237,6 +237,21 @@ def _get_style_of_faulting_term(C, ctx):
     res[(ctx.rake > 30.) & (ctx.rake < 150.)] = C["e3"]
     return res
 
+def get_epistemic_sigma(ctx):
```

Comment:
If I am not wrong this code is exactly the same added to ask14. Instead of repeating it here, wouldn't it be better to import it? Ideally, this correction of the mean due to epistemic sigma could be implemented in the modifiable GMPE.